src/network_analysis.jl

```diff
@@ -436,18 +487,23 @@ subnetworks(sir)
 """
 function subnetworks(rs::ReactionSystem)
     isempty(get_systems(rs)) || error("subnetworks does not currently support subsystems.")
+
+    # Retrieves required components. `linkageclasses` caches linkage classes in `rs`.
     lcs = linkageclasses(rs)
     rxs = reactions(rs)
     p = parameters(rs)
     t = get_iv(rs)
     spatial_ivs = get_sivs(rs)
     complextorxsmap = [map(first, rcmap) for rcmap in values(reactioncomplexmap(rs))]
     subnetworks = Vector{ReactionSystem}()
+
+    # Loops through each sub-graph of connected reaction complexes. For each, create a 
+    # new `ReactionSystem` model and pushes it to the subnetworks vector.
     for i in 1:length(lcs)
-        reacs, specs, newps = subnetworkmapping(lcs[i], rxs, complextorxsmap, p)
```

Comment:
Here, I found it weird that `rxs` was used for the old reactions, and `reacs` was used for the new reactions (while `newps` and `p` was used for the new/old parameters). In this, and the previous `subnetworkmapping` function, I propose we change to `newrxs`, `newspecs`, and `newps` for the new reactions, species, and parameters.